packages/katello/katello-repos/katello.repo

```diff
@@ -11,9 +11,15 @@ gpgcheck=0
 name=Pulp Community Release
 baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/@PULP_RELEASE@/@PULP_VERSION@/$releasever/$basearch
 gpgkey=https://repos.fedorapeople.org/repos/pulp/pulp/GPG-RPM-KEY-pulp-2
-enabled=1
+enabled=@ENABLE_PULP_STABLE@
 gpgcheck=1
 
+[pulp-nightly]
+name=Pulp Nightly Community Release
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/testing/automation/2-master/stage/$releasever/$basearch
+enabled=@ENABLE_PULP_NIGHTLY@
+gpgcheck=0
```

Comment:
having pulp-nightly in the repo file at all times seems not ideal, but maybe its not a huge deal.  @ehelms since you've worked in this file quite a bit, i'm curious your thoughts?

Reply:
@ehelms any thoughts on this?

Reply:
I'd prefer to set the URL in a variable in the spec then to sed that in. Not sure if it would be prettier.

Reply:
I made it easy to switch between the two in the repos file in case we want to keep the nightly entry, but I think we can remove the nightly repo entry completely when we switch to 2.10 beta since this is a temporary thing and not how we usually work with pulp releases.

Reply:
I think I'd prefer to do that part in the spec file and not in the actual file. So we'd have `baseurl=@PULP_URL@` and also `gpgcheck=@PULP_GPGCHECK`.

Reply:
We may need to go this nightly route again in the future so having it future proofed would be useful.